modules/dnn/src/layers/max_unpooling_layer.cpp

```diff
@@ -110,17 +110,17 @@ class MaxUnpoolLayerImpl CV_FINAL : public MaxUnpoolLayer
                     int index = idxptr[i_wh];
                     if (!(0 <= index && index < outPlaneTotal))
                     {
-                        std::cerr
-                            << "i_n=" << i_n << std::endl
-                            << "i_c=" << i_c << std::endl
-                            << "i_wh=" << i_wh << std::endl
-                            << "index=" << index << std::endl
-                            << "maxval=" << inptr[i_wh] << std::endl
-                            << "outPlaneTotal=" << outPlaneTotal << std::endl
-                            << "input.size=" << input.size << std::endl
-                            << "indices.size=" << indices.size << std::endl
-                            << "outBlob=" << outBlob.size << std::endl
-                            ;
```

Comment:
This one is useful from time to time. So if you could find a way for multiline `CV_LOG` that would be nice.